Source/KSCrash/Recording/Monitors/KSCrashMonitor_MachException.c

```diff
@@ -274,7 +274,7 @@ static void* handleExceptions(void* const userData)
 
     const char* threadName = (const char*) userData;
     pthread_setname_np(threadName);
-    if(threadName == kThreadSecondary)
+    if(strcmp(threadName, kThreadSecondary))
```

Comment:
This (unintentionally?) reverses the logic... maybe it should be `if (!strcmp(threadName, kThreadSecondary))` ?

Reply:
oh nice catch, thank you!